tests/spark/test_mrjob_spark_harness.py

```diff
@@ -253,7 +265,7 @@ def test_compression(self):
             runner.run()
 
             self.assertTrue(runner.fs.exists(
-                join(runner.get_output_dir(), 'part*.gz')))
+                os.path.join(runner.get_output_dir(), 'part*.gz')))
```

Comment:
You don't like `from os.path import join`?

Reply:
I just needed other methods from `os.path` so I switched to import `os.path` as a whole -- I can import the individual methods I'm using if you'd prefer.

Reply:
mrjob isn't totally consistent about this, but my preference is to import the individual function names (and I occasionally change modules over to this way of doing things). I just find it really noisy to read nested calls to various `os.path.*()` functions.

Reply:
My standard is use `from X import Y` unless `Y` is a vague name that needs `X` for context. Which admittedly is a bit squishy.

Reply:
ok, fixed.